### PR TITLE
Added a list of abbreviations with meanings to the High Throughput Molecular Dynamics and Analysis Tutorials (Computational Chemistry) in response to feedbacks

### PR DESCRIPTION
**Pull request by Ezichi-Fav**
I updated the contributors.yaml file and  Added a list of abbreviations with meanings to the High Throughput Molecular Dynamics and Analysis Tutorials (Computational Chemistry) in response to feedbacks.

### CODE REVIEW

**Comment by bebatut** [comment]
Thanks @Ezichi-Fav.
@simonbray could you have a look at this PR? Thanks a lot

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 1]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
 >
 {: .agenda}
 
+
+# Abbreviations
+* **Hsp90** –Heat Shock Protein 90 <br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:46]:
```suggestion
* **Hsp90** - Heat Shock Protein 90
```
Can you make sure you format the same on each line? With 1 space before and after the dash.

Also, the `<br>` tag should not be needed.

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 2]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **MD**- Molecular Dynamics Simulation<br>
+* **PDB**- Protein Data Bank<br>
+* **GROMACS**- Groningen Machine for Chemical Simulation. It is a molecular dynamics package mainly designed for simulations of proteins, lipids and nucleic acids. <br>
+* **GRO files**- GROMAC Format<br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:50]:
```suggestion
* **GRO files**- structure file in GROMACS format
```

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 3]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **DCD** - Dynamics CoorDinate files<br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:51]:
```suggestion
* **DCD** - trajectory format supported by CHARMM and NAMD (Dynamics CoorDinate)
```

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 4]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **RSMD**-Root Mean Square Deviation<br>
+* **RSMF**- Root Mean Square Fluctuation<br>
+* **PCS**- Principal Components analysis<br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:54]: Please can you check spelling of these 3 abbreviations here?

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 5]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **NTD**- N-Terminal Domain. <br>
+* **ATP**- Adenosine Triphosphate<br>
+* **TRR**- trajectory file (coordinates and velocity) <br>
+* **XTC**- trajectory file (coordinates only) <br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:58]: mention here TRR and XTC are supported by GROMACS

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 6]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **ACPYPE tool**- AnteChamber Python Parser interface<br>
+* **GROMOS**- GROningen Molecular Simulation<br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:60]:
```suggestion
* **GROMOS**- force field (GROningen Molecular Simulation)
```
can you make the same change for the next three force fields as well?

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 7]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **CHARMM**- Chemistry at Harvard Macromolecular Mechanics<br>
+* **AMBER**- Assisted Model building with Energy Refinement<br>
+* **OpenFF**-Open Force Field<br>
+* **TIP3P**- A water model that specifies rigid water molecules with charges and Lennard-Jones Paramenters assigned to each of the 3 atoms<br>
+* **ITP files**- Atom Topologies (charges, mass, radii etc) <br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:65]: mention it is a GROMACS file format

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 8]
```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
...
+* **GAFF**- (General AMBER  Force Field) <br>
+* **NVT**- constant number (N), Volume (V) Temperature (T) <br>
+* **NPT**- constant number (N) , Pressure (P), Temperature (T) <br>
+* **NMR**- Nuclear Magnetic Resonance<br>
+* **VMD**- Visual Molecular Dynamics <br>
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:70]:
```suggestion
* **VMD**- Visual Molecular Dynamics, a visualization software
 ```

**Review by simonbray — COMMENTED** [review]
Hi @Ezichi-Fav, thanks a lot for the contribution! I wrote a few comments. 

In general, I think it's important that the users understand what the abbreviations mean, not only what they stand for, so I suggested a few places where some more detail could be added. For example, to know what CHARMM stands for is not as important as to know what it actually is (a force field). :)

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
thanks @Ezichi-Fav!

**Pull request merged**